turbinia/evidence.py

```diff
@@ -513,3 +513,11 @@ def __init__(self, module_list=None, profile=None, *args, **kwargs):
     super(RawMemory, self).__init__(*args, **kwargs)
     self.profile = profile
     self.module_list = module_list
+
+
+class BinaryExtraction(Evidence):
```

Comment:
See below for more details, but you might need to extend a different Evidence type here.  If you only want the log file and not all of the binaries you might consider changing this to something like BinaryHashList instead.

Reply:
I want to have both (binaries + file with all the hashes), see my other comment for more details.